src/Symfony/Component/Encryption/Ciphertext.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption;
+
+use Symfony\Component\Encryption\Exception\DecryptionException;
+use Symfony\Component\Encryption\Exception\MalformedCipherException;
+
+/**
+ * This class is responsible over the payload API.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.3
+ *
+ * @internal
```

Comment:
It is only experimental if it is public, I think it is not needed to mark it as experimental if it is internal and vice versa 🤔